java/src/apps/Apps.java

```diff
@@ -263,7 +264,13 @@ public boolean execute() {
         if (file.exists()) {
             log.debug("start load config file {}", file.getPath());
             try {
-                configOK = InstanceManager.getOptionalDefault(jmri.ConfigureManager.class).load(file, true);
+                ConfigureManager cmOD = InstanceManager.getOptionalDefault(jmri.ConfigureManager.class);
+                if (cmOD != null) {
+                    configOK = cmOD.load(file, true);
```

Comment:
I'm not sure of the reasoning why there are two of them: getDefault and getOptionalDefault. But I've got another 55 files where I worked on protecting getOptionalDefault.

At least now the tests passed.